addscore.php

```diff
@@ -0,0 +1,69 @@
+<?php
+include "classes.php";
+session_start();
+
+include "commonUI.php";
+$scores = array();
+startHTML("scores");
+
+
+$health = $_SESSION["playParty"]->_health;
+$members = $_SESSION["playParty"]->_livingMembers;
+
+if(strcmp($health, "Good") == 0){$scores[0] = 500 * $members;}
```

Comment:
Before this, you should call the `health()` function of the party to make sure the `_health` member is up-to-date.